### PR TITLE
Description #65

### PR DESCRIPTION
**Pull request by NisanAyash**

### CODE REVIEW

**Comment by urish** [comment]
Thanks! I suggest changing the subtitle to bullets, e.g.:

<div dir='rtl' align='right'>

רוצה לקחת חלק?

* מדי חודש אנו עורכים ערבי סדנאות מונחים לתרומה לקוד פתוח  
* ניתן להתעדכן על הסדנאות ועל הרצאות העשרה שאנו מארגנים בשבילכם בקבוצת הטלגרם (לינק)
* בקבוצת הפייסבוק (לינק) ניתן לשאול שאלות ולשתף מהידע שלכם בנושאי תרומה לקוד פתוח
* להצטרפות לצוות המנטורים שלנו (לינק לטופס)  

</div>

File: components/description/description.module.scss [hunk 1]
```diff
@@ -0,0 +1,58 @@
+$colors: (
+  one: #456c3d,
+  two: #669f54,
+  three: #7ec16a,
+  four: #b9e7ac,
+  five: #ecedf0,
+);
```
Comment (MichalPorag) [inline components/description/description.module.scss:7]: Please use sass array insted.

File: components/description/description.module.scss [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    width: 3vmin;
+    height: 3vmin;
+    background: darken($color, 0%);
+    border-radius: 5px;
+    box-shadow: 0 0 5px 1px lighten($color, 25%);
+    border: 5px solid darken($color, 5%);
+    margin: 15px 0;
+    transform: rotate(-90deg);
+  }
+}
```
Comment (MichalPorag) [inline components/description/description.module.scss:20]: Update according to the sass array change.

File: components/description/Description.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+          אם אתם רוצים להצטרף כמנטורים של פרויקט ,<br /> מוזמנים למלא את הטופס הבא:
+          <a href="https://forms.gle/6mHgTs5PnsfXUZ1f8">✍</a>
+        </p>
+      </div>
+      <ul className={style.squareContainer}>
+        <li className={style.one} id="1"></li>
+        <li className={style.two} id="2"></li>
+        <li className={style.three} id="3"></li>
+        <li className={style.four} id="4"></li>
+        <li className={style.five} id="5"></li>
```
Comment (MichalPorag) [inline components/description/Description.tsx:28]: No need for the class name and the ids.

File: components/layout/layout.tsx [hunk 4]
```diff
@@ -22,10 +22,9 @@ const Layout = ({ children }: LayoutProps) => {
       <style>{`
                 .layout__container {
                     margin: 0 auto;
-                    padding: 0 25px;
                 }
                 .layout__body--container {
-                    margin-top: 70px;
+                    margin-top: 0px;
                 }
```
Comment (MichalPorag) [inline components/layout/layout.tsx:28]: Please remove that.

File: components/layout/layout.tsx [hunk 5]
```diff
@@ -22,10 +22,9 @@ const Layout = ({ children }: LayoutProps) => {
       <style>{`
                 .layout__container {
```
Comment (MichalPorag) [inline components/layout/layout.tsx:23]: Please clean the class of this html so only one ".layout__container" will stay.

**Review by MichalPorag — CHANGES_REQUESTED** [review]

**Comment by MichalPorag** [comment]
@NisanAyash 
Great job!! And well done on the sass loop challenge 💪🏻
Just one problem, this is how it is look between 1420-1459 view width, and a few suggestion on the code :) 

image

File: components/layout/layout.tsx (hunk 5 above)
Comment (NisanAyash) [reply components/layout/layout.tsx:23]: > @NisanAyash
> Great job!! And well done on the sass loop challenge 💪🏻
> Just one problem, this is how it is look between 1420-1459 view width, and a few suggestion on the code :)
> 
> image

Hey, @MichalPorag Thanks for your comments...
I'll fix that and send PR again soon.

File: components/description/Description.tsx (hunk 3 above)
Comment (NisanAyash) [reply components/description/Description.tsx:28]: You actually right!

File: components/layout/layout.tsx (hunk 4 above)
Comment (NisanAyash) [reply components/layout/layout.tsx:28]: And we talked about that

File: components/description/description.module.scss (hunk 1 above)
Comment (NisanAyash) [reply components/description/description.module.scss:7]: Example for sass aray
```scss
$sizes: 40px, 50px, 80px;

@each $size in $sizes {
  .icon-#{$size} {
    font-size: $size;
    height: $size;
    width: $size;
  }
}
```

File: components/description/description.module.scss [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+.container {
+  width: 100%;
+  min-height: 45vh;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 4vmin;
```
Comment (MichalPorag) [inline components/description/description.module.scss:25]: Please don't use a vmin unit on fonts. 
In big screen it is not looking good.
Instead, give a the root a difficult text size and use rem units. If you have a nested text use em unit.

File: components/description/description.module.scss [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+    .squareContainer {
+      transform: rotate(90deg);
+      margin: 0 auto;
+    }
+  }
+}
+
+.brandColor {
+  color: #ce5a55;
+}
```
Comment (MichalPorag) [inline components/description/description.module.scss:45]: Please remove and get the color from the colors file, https://github.com/urish/pull-request-community/blob/master/styles/colors.ts

File: components/description/description.module.scss [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+.container {
+  width: 100%;
+  min-height: 45vh;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 4vmin;
+  font-weight: 300;
+  font-family: Assistant;
```
Comment (MichalPorag) [inline components/description/description.module.scss:27]: No need, it was already declared in the global file: https://github.com/urish/pull-request-community/blob/master/styles/globals.scss

File: components/description/description.module.scss [hunk 9]
```diff
@@ -0,0 +1,55 @@
+$colors: #456c3d, #669f54, #7ec16a, #b9e7ac, #ecedf0;
```
Comment (MichalPorag) [inline components/description/description.module.scss:1]: Please add this array to the colors file and import it from there: https://github.com/urish/pull-request-community/blob/master/styles/colors.ts

File: components/description/Description.tsx [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+          אם אתם רוצים להצטרף כמנטורים של פרויקט ,<br /> מוזמנים למלא את הטופס הבא:
+          <a href="https://forms.gle/6mHgTs5PnsfXUZ1f8">✍</a>
+        </p>
+      </div>
+      <ul className={style.squareContainer}>
+        <li></li>
+        <li></li>
+        <li></li>
+        <li></li>
+        <li></li>
```
Comment (MichalPorag) [inline components/description/Description.tsx:28]: Please use map to right that dynamically.

**Review by MichalPorag — CHANGES_REQUESTED** [review]
Hi! great job and thank you so match! 
A few changes needed, and please check the UI bug that reproduce if  the screen size is between 1450px to 1285px.

**Review by MichalPorag — APPROVED** [review]

**Pull request merged**